circle.yml

```diff
@@ -149,20 +163,23 @@ commands:
         default: 477
 
     steps:
+      - install_wabt
       - run:
           name: "Download spectest files"
           working_directory: ~/build
           command: |
             if [ ! -d wasm-spec ]; then
-              git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-jsontests-20200813 --depth 1
+              git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-jsontests-20200813 --depth 1 wasm-spec
+              mkdir json && cd json
+              find ../wasm-spec/test/core -name '*.wast' -exec wast2json {} \;
```

Comment:
`wast2json` has the same flags for disabling extensions, I think it's better to use them here, too.

Reply:
Done.